conans/util/files.py

```diff
@@ -158,7 +158,7 @@ def mkdir_tmp():
 def to_file_bytes(content):
     if six.PY3:
         if not isinstance(content, bytes):
-            content = bytes(content, "utf-8")
+            content = bytes(content, "utf-8", "ignore")
```

Comment:
Should we ignore this or as we don't know how to properly write the fail it is ok to fail?

Reply:
Ignore should work for this case, but I'm afraid about possible side effects. I would prefer write a good message error, as this error is very specific. WDYT?

Reply:
Yes, this is problematic, just ignoring and dropping characters because an encoding issue is not an option.